docs/dev/android/gradle-plugin-in-mc.md

```diff
@@ -0,0 +1,35 @@
+# Replacing the Glean Gradle plugin in mozilla-central
+
+> **Note**: If you only need to replace the `glean_parser` Python parts used in the build see [Substituting `glean_parser`](glean-parser-substitution.md).
+> The approached documented in this chapter is only necessary if you changed `GleanGradlePlugin.groovy`.
+
+If you need to replace the Glean Gradle Plugin used by any component in mozilla-central, follow these steps:
+
+1. In your Glean repository increment the version number in `.buildconfig.yml` to something unused:
+
+   ```yaml
+   libraryVersion: 70.0.0
+   ```
+
+1. Build the Glean Gradle plugin and publish the plugin locally:
```

Comment:
Missing requirement of needing to have maven installed locally to serve the package. I think you can get it from `homebrew` 🤔